features/step_definitions/check_basecap.rb

```diff
@@ -0,0 +1,18 @@
+require 'open3'
+require 'json'
+
+def is_capability_enabled?(capability)
+  cmd = 'oc get clusterversion -o=jsonpath={.items[*].status.capabilities.enabledCapabilities}'
+  stdout, stderr, status = Open3.capture3(cmd)
```

Comment:
@dpuniaredhat Like lib/environment.rb:315:
```
res = admin.cli_exec(:get, resource: "clusterversion", resource_name: "version", o: "jsonpath={.status.desired.version}")
```
You can use similarly.

Reply:
@xingxingxia I tried adding `admin.cli_exec` but observed that this method sets its own config file, which caused the failure below:
https://privatebin.corp.redhat.com/?fe75eaf7b2d9af41#2toPmeRUScXF9n6w2LKvaneDUY8ALqBZMQfZeq2ueii4

I think using `Open3.capture3` is also fine if it doesn't have issues. for me It will take more time for me to analyze `admin.cli_exec`.

Reply:
Could you run the test with Runner job, to make sure it works in Jenkins CI ?

Reply:
@dpuniaredhat the framework already has support for get openshift resources and parse out the output for properties you are looking for.  Do you don't need to create this new method with `Open3`.  Just expand on the exisiting [cluster_version.rb class ](https://github.com/openshift/verification-tests/blob/master/lib/openshift/cluster_version.rb)

Reply:
@dpuniaredhat @liangxia @xingxingxia I've submitted a PR #3624 that we can use to check the `clusterversion` capabilities.